src/SocialLogin.android.ts

```diff
@@ -179,9 +179,8 @@ export class SocialLogin extends Social {
             throw e;
         }
     }
-    logoutWithGoogle(callback: (result: Partial<ILoginResult>) => void) { 
-        if (this._googleClient.isConnected())
-        {
+    logoutWithGoogle(callback: (result: Partial<ILoginResult>) => void) {

```

Comment:
@govi2010 usually though, when a new feature is added, its a requirement its implemented across both platforms. This seems like a pretty neat feature but pending when i could collaborate on this new feature, could you revert this feat on the PR so we can create a new PR for it later on so can it can be replicated across board for both platforms and for all providers?